redbot/cogs/customcom/customcom.py

```diff
@@ -206,6 +206,24 @@ async def customcom(self, ctx: commands.Context):
         """Custom commands management."""
         pass
 
+    @customcom.command(name="raw")
+    async def cc_raw(self, ctx: commands.Context, command: str.lower):
+        """Get the raw response of a custom command, to get the proper markdown.
+        
+        This is helpful for copy and pasting."""
+        commands = await self.config.guild(ctx.guild).commands()
+        if not command in commands:
```

Comment:
You should use `x not in y` over `not x in y`:
```suggestion
        if command not in commands:
```